data_visualization_with_matplotlib_and_seaborn.ipynb

```diff
@@ -138,12 +138,12 @@
     "import pandas as pd\n",
     "\n",
     "# Link to the GitHub-hosted files\n",
-    "data_m = \"https://raw.githubusercontent.com/MrGeislinger/flatiron-school-data-science-curriculum-resources/main/DataScienceBasics/Visualization/data/data9b_m.txt\"\n",
-    "data_w = \"https://raw.githubusercontent.com/MrGeislinger/flatiron-school-data-science-curriculum-resources/main/DataScienceBasics/Visualization/data/data9b_w.txt\"\n",
+    "data_A = \"https://raw.githubusercontent.com/MrGeislinger/flatiron-school-data-science-curriculum-resources/main/DataScienceBasics/Visualization/data/data9b_m.txt\"\n",
```

Comment:
Since data is fake, it might be better to stay away from BMI & steps. Should change to something more general